src/main/java/org/spongepowered/server/launch/VanillaServerMain.java

```diff
@@ -30,18 +30,23 @@
 import static org.spongepowered.server.launch.VanillaCommandLine.TWEAK_CLASS;
 import static org.spongepowered.server.launch.VanillaCommandLine.VERSION;
 
+import com.google.gson.JsonElement;
```

Comment:
This is a problem: GSON isn't bundled in the SpongeVanilla JAR and is therefore not available before the MC server JAR was downloaded. I guess we'll now need to shade it to parse that JSON file...

Reply:
Maybe use JSON.simple instead for a lightweight dependency?